themes/sensei-course-theme/templates/lesson.4-0-2.html

```diff
@@ -22,7 +22,7 @@
         <!-- /wp:column -->
     </div>
     <!-- /wp:columns -->
-    <!-- wp:sensei-lms/course-theme-course-progress-bar /-->
+		<!-- wp:sensei-lms/course-theme-course-progress-bar {"style":{"spacing":{"margin":{"right":"0","left":"0","top":"0","bottom":"0"}}}} /-->
```

Comment:
I'm not too sure what this template is used for (Sensei 4.0.2 version only maybe?), but I believe the correct template file is the one in the `templates\default` folder.

Reply:
Hum, but it was working :)